plugins/plugin-nodejs-debugger/che-plugin-nodejs-debugger-server/src/main/java/org/eclipse/che/plugin/nodejsdbg/server/NodeJsDebugger.java

```diff
@@ -115,7 +116,7 @@ public void addBreakpoint(Breakpoint breakpoint) throws DebuggerException {
             Location location = breakpoint.getLocation();
             pendingBreakpoints.add(location);
             library.setBreakpoint(location.getTarget(), location.getLineNumber());
-            checkActivatedBreakpoints();
+            debuggerCallback.onEvent(new BreakpointActivatedEventImpl(breakpoint));
```

Comment:
Since you immediately send  `BreakpointActivatedEventImpl` no need to check if they are activated later.